scripts/system/create/edit.js

```diff
@@ -2905,7 +2905,21 @@ function getExistingZoneList() {
         };
         listExistingZones.push(thisZone);
     }
+    listExistingZones.sort(zoneSortOrder());
```

Comment:
```suggestion
    listExistingZones.sort(zoneSortOrder);
```
Just need to pass in the function identify, not call it.